blankly-finance/blankly
# slate dedup

- initial draft of futures api and binance implementation
- add example
- add binance futures order implementation
- v1.16.2-beta release
- binance futures impl
- Remove strategy export from reporter headers & strategy
- v1.16.3-beta release
- v1.16.4-beta release
- Add tokenbot sponsorship
- v1.16.5-beta release
- Remove quantstats conversion in examples
- Added FTX tld support
- Added ftx_tld in example/settings.json
- add functionality to binance futures exchange
- refactor futures code
- save
- Update tests for better order tracking
- Add trunc to tests
- Add ftx tld to default settings
- Correct cancel order #134
- Always use the base asset when using .get_account
- improve testing and ftx interface
- ftx funding rates
- Add backtesting in platform & auth caching in CLI
- Fix metrics bug, make post accurate
- Add better terminos handling in CLI & a backtest status post
- fix funding rate and add order methods
- implement order functions ftx futures
- return precision for products
- Clean platform export
- supercharged testing
- Allow the model to be attached to an existing platform model & improve index maxes/mins. Delete the blankly.json if something goes wrong.
- add tests
- v1.16.6-beta release
- v1.16.7-beta release
- v1.16.8-beta release
- v1.16.8-beta release
- v1.16.9-beta release
- v1.16.9-beta release
- Add logout function and make compatible with alpaca V2 API
- v1.16.10-beta release
- test test test
- Fix alpaca history function
- v1.16.11-beta release
- pass all tests
- fix binance bugs and add more tests
- make minor changes to tests
- fix typing errors and move things to futures
- remove comment
- add copyright comments
- Add print in backtest
- small fixes
- fix test for py3.7
- run formatter
- hehe
- Minor import changes
- merge
- fix tests
- fix tests
- fix tests again
- fix lru_cache issue
- remove old binance_futures tests
- Modify parser import
- Fix tests & add backtest view link
- Cancel market order if market isn't open
- v1.16.12-beta release
- fix cagr metric
- add new cli
- add commands
- fix
- add fun spinner
- new cli work
- finish new cli
- bugfix
- remove debug print
- teams fixes
- teams
- don't show team if not needed
- asdf
- ffff
- work on localhost
- fix bugs
- remove debug
- gen keys.json
- fix bugs w teams stuff
- add file check
- add doccomments and small fixes
- replace imports
